lib/shopify_transporter/pipeline/magento/order/top_level_attributes.rb

```diff
@@ -104,7 +100,11 @@ def total_refunded(hash)
           end
 
           def paid?(hash)
-            total_price(hash) == total_paid(hash) && total_refunded(hash) == 0
+            total_price(hash) == total_paid(hash) && total_paid(hash) > 0 && total_refunded(hash) == 0
+          end
+
+          def unpaid?(hash)
```

Comment:
This is no longer used and we can remove it.

Reply:
^ 👍

Reply:
Will do